harness/tests/fixtures/trial_logs.json

```diff
@@ -0,0 +1,49 @@
+{"result":{"id":"138905","timestamp":"2023-07-06T23:56:35.680736Z","message":"[2023-07-06T23:56:35.680736Z]          || INFO: Scheduling Trial 308 (Experiment 313) (id: f0804fb9-c627-4d68-85d6-abfe020ba325)\n","level":"LOG_LEVEL_INFO","trialId":308,"log":"Scheduling Trial 308 (Experiment 313) (id: f0804fb9-c627-4d68-85d6-abfe020ba325)\n","source":"master","stdtype":"stdout"}}
```

Comment:
hm, well, this file isn't technically a properly-formatted json. it contains a bunch of JSON objects separated by newlines, because this is what the REST API effectively does. if i prettify it, the newline-delimiting won't work anymore.

Reply:
Oh! I hadn't read the code closely enough to understand the call. Small request, then, that is possible overly pedantic: `sample_get_trial_logs` -> `sample_trial_logs` and possibly relocating this method (no worries, we can deal with it later if nothing comes to mind), since this doesn't fake a response from `bindings` like the rest of the code.